frontend/src/static/js/v_summary.js

```diff
@@ -123,16 +123,20 @@ window.vSummary = {
   watch: {
     sortGroupSelection() {
       this.getFiltered();
+      deactivateAllOverlays();
```

Comment:
instead of placing `deactivateAllOverlays()` in all watchers, how about place it within `getFiltered()`?

Reply:
There is a watcher that calls `getFiltered()` but does not require the overlays to be removed, e.g. `filterBreakdown()`. What do you suggest in that case?

Reply:
I guess for simplicity's sake, can just remove the overlays as well? @fzdy1914 your inputs?

Reply:
Hmmm, I believe it is better to put it inside `getFiltered()`. 

For control like `filterBreakdown`, after looking at the source code of how this flag works. This watcher seems unnecessary. You may have an investigation about it. @bluein-green

Reply:
I tried removing the watcher for `filterBreakdown`. In doing so, the breakdown functionality still works. However, as `setSummaryHash()` is only called from `getFiltered()`, the hash would not immediately be set to `breakdown=true` if `filterBreakdown = false` initially. (the hash _would_ contain `breakdown=true` when changing other settings that result in `getFiltered()` being called)

What do you think about putting `deactivateAllOverlays()` in `getFiltered()`, retaining the watcher for `filterBreakdown` and calling `setSummaryHash()` in the watcher? It seems to work when I tried it.

Reply:
I suggest, putting `deactivateAllOverlays()` in `getFiltered()`, remove the watcher for `filterBreakdown` and add the hash directly in the `filterBreakdown`.

Reply:
I believe we can still have the watcher for `filterBreakdown`, but to change the `getFiltered()` to `setSummaryHash()`.